apps/customer-portal/backend-v2/internal/handler/caller_scope_test.go

```
@@ -0,0 +1,971 @@
+		req := authedRequest(http.MethodGet, "/conversations/11111111-1111-1111-1111-111111111111/messages", "")
+		rec := httptest.NewRecorder()
+		mux.ServeHTTP(rec, req)
+
+		if rec.Code != http.StatusNotFound {
+			t.Fatalf("expected 404, got %d: %s", rec.Code, rec.Body.String())
+		}
+	})
+}
+
+type fakeEntityUserProjectClient struct {
+	entityUserProjectClient
```

golangci-lint (2.12.2)

[error] field entityUserProjectClient is unused

(unused)